packages/amplify_datastore/example/ios/unit_tests/resources/modelSchema/model_schema_maps.json

```diff
@@ -1,264 +1,279 @@
+                    "ofModelName": "Post"
+                },
+                "isRequired": false,
+                "isArray": false,
+                "association": {
+                    "associationType": "BelongsTo",
+                    "targetName": "postID"
+                }
+            },
+            "content": {
+                "name": "content",
+                "type": {
+                    "fieldType": "string"
+                },
+                "isRequired": true,
+                "isArray": false
+            }
+        }
+    },
+
+    "PostSchema": {
+        "name": "Post",
+        "pluralName": "Posts",
+        "fields": {
+            "id": {
+                "name": "id",
+                "type": {
+                    "fieldType": "string"
+                },
+                "isRequired": true,
+                "isArray": false
+            },
+            "title": {
+                "name": "title",
+                "type": {
+                    "fieldType": "string"
+                },
+                "isRequired": true,
+                "isArray": false
+            },
+            "created": {
+                "name": "created",
+                "type": {
+                    "fieldType": "dateTime"
+                },
+                "isRequired": false,
+                "isArray": false
+            },
+            "rating": {
+                "name": "rating",
+                "type": {
+                    "fieldType": "int"
+                },
+                "isRequired": false,
+                "isArray": false
+            },
+            "blog": {
+                "name": "blog",
+                "type": {
+                    "fieldType": "model",
+                    "ofModelName": "Blog"
+                },
+                "isRequired": false,
+                "isArray": false,
+                "association": {
+                    "associationType": "BelongsTo",
+                    "targetName": "blogID"
+                }
+            },
+            "comments": {
+                "name": "comments",
+                "type": {
+                    "fieldType": "collection",
+                    "ofModelName": "Comment"
+                },
+                "isRequired": false,
+                "isArray": true,
+                "association": {
+                    "associationType": "HasMany",
+                    "associatedName": "post",
+                    "associatedType": "Post"
+                }
+            },
+            "author": {
+                "name": "author",
+                "type": {
+                    "fieldType": "model",
+                    "ofModelName": "Author"
+                },
+                "isRequired": true,
+                "isArray": false,
+                "association": {
+                    "associationType": "BelongsTo",
+                    "targetName": "authorId"
+                }
+            }
+        }
+    },
```

Comment:
Converted this file to LF mode to align with other files in the codebase. The actual change is to add the `author` data